tests/job/test_genericJob.py

```diff
@@ -4,6 +4,8 @@
 
 import unittest
 import os
+from time import sleep
+from concurrent.futures import ProcessPoolExecutor
```

Comment:
```suggestion
from concurrent.futures import Future, ProcessPoolExecutor
```

Necessary for copy test suggestion below